src/PowerShellEditorServices/Services/TextDocument/ScriptRegion.cs

```diff
@@ -12,64 +14,42 @@ namespace Microsoft.PowerShell.EditorServices.Services.TextDocument
     /// </summary>
     public sealed class ScriptRegion : IScriptExtent
     {
-        #region Static Methods
+        public TextEdit ToTextEdit() => new() { NewText = Text, Range = ToRange() };
```

Comment:
```suggestion
        internal TextEdit ToTextEdit() => new() { NewText = Text, Range = ToRange() };
```

This class is public and surfaced, I'd recommend keeping these internal unless you were explicitly intending on providing it as an extension point